### PR TITLE
List on Artifacthub: add initial artifacthub-repo.yml to claim repository ownership

### PR DESCRIPTION
**Pull request by reta**
Part of https://github.com/openzipkin/zipkin-helm/issues/21

### CODE REVIEW

File: artifacthub-repo.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+owners:
```
Comment (reta) [inline artifacthub-repo.yml:1]: @bornav @codefromthecrypt seems like this is a first step, with that we should be able to claim repository ownership and I could get repository ID

**Review by codefromthecrypt — APPROVED** [review]
approve despite CI unrelated fail

**Comment by reta** [comment]
> approve despite CI unrelated fail

Thanks @codefromthecrypt , I will try to fix the CI shortly, if you don't mind, I will merge this one but there would be more changes (repositoryID, etc), thanks!

**Comment by codefromthecrypt** [comment]
thanks for the help and follow-up @reta!

File: artifacthub-repo.yml (hunk 1 above)
Comment (bornav) [reply artifacthub-repo.yml:1]: here is the id of the one I created, if you can't transfer it to your account I would recommend creating a new one
071a8849-26f8-4d02-b8e2-898966d5a1a7

**Pull request merged**